awscli 1.17.7

temal- opened the pull request:
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

`brew audit --strict awscli` fails with:

```
awscli:
  * should only be updated every 10 releases on multiples of 10
Error: 1 problem in 1 formula detected
```

As described in #49095, the update is needed because awscli 1.17.0 generates multiple lines of output in the form of the following line whenever invoked:

```
/usr/local/Cellar/awscli/1.17.0/libexec/lib/python3.8/site-packages/jmespath/visitor.py:32: SyntaxWarning: "is" with a literal. Did you mean "=="? if x is 0 or x is 1:
```

Thanks!

bayandin commented:
As far as I know, this warning is not fixed in 1.17.7, see https://github.com/aws/aws-cli/issues/4833

temal- commented:
Both linked sources do not mention 1.17.7. Only 1.17.0.

I tried a couple of commands (`s3 sync`, `dynamodb list-tables`, `sts assume-role`) and cannot see any more SyntaxWarning with 1.17.7 at this point.

bayandin commented:
You're probably right, I just pointed out that the issue still open.
Btw, **1.17.8** has been released.

temal- commented:
> You're probably right, I just pointed out that the issue still open.

I added a comment to the ticket in the awscli repo. Let's see what happens. 

> Btw, 1.17.8 has been released.

😱
I added another commit. :)

chenrui333 commented:
As daily user, I will take it.

chenrui333 commented:
Thanks @temal- for efforts on this!!
